packages/elemental-theme/src/custom-elements/ef-text-field.less

```diff
@@ -32,4 +32,12 @@
   &[disabled] [part='input'] {
     user-select: none;
   }
+
+  [part~='button'] {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    flex: none;
+  }
```

Comment:
missing hover style for text, number, password, email and search field.

Reply:
I observed that the color of hovering icon doesn't change in Elemental theme. So I doesn't style it.